VizbeeSmartPromptOptions.js

```diff
@@ -0,0 +1,13 @@
+export default class VizbeeSmartPromptOptions {
+    
+    constructor() {
+        this.enabledSubflows = 
+        SmartPromptSubflow.CastAuthorization | SmartPromptSubflow.CastIntroduction | SmartPromptSubflow.SmartInstall;
+    }
+}
+
+export const SmartPromptSubflow = {
```

Comment:
Can you let me know how will the user use this to set the sub-flows?

Reply:
```
import {VizbeeSmartPromptOptions, SmartPromptSubflow } from 'react-native-vizbee-sender-sdk';

const smartPromptOptions = new VizbeeSmartPromptOptions();
    smartPromptOptions.enabledSubflows = SmartPromptSubflow.CastIntroduction | SmartPromptSubflow.SmartInstall
    VizbeeManager.smartPromptWithOptions(smartPromptOptions);
```